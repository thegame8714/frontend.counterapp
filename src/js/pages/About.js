import React, { Component } from "react";
import PageHeading from '../components/PageHeading';


class About extends Component {

	/**
	 * Creates an instance of About.
	 * 
	 * @param {any} props
	 * 
	 * @memberOf About
	 */
	constructor(props) {
		super()
	}


	/**
	 * 
	 * 
	 * @returns
	 * 
	 * @memberOf About
	 */
	render() {
		return (
			<div className='content'>
				<PageHeading value="About page" />
			</div>
		);
	}
}

export default About;
