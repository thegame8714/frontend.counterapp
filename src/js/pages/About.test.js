import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import About from './About';

describe('About Page', () => {
  it('should render page tile as H1 with text', () => {
    const route = { pageTitle: "Test Page title" }
    const wrapper = mount(<About route={route} />);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});