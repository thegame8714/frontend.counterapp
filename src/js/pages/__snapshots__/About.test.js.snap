exports[`About Page should render page tile as H1 with text 1`] = `
<About
  route={
    Object {
      "pageTitle": "Test About page title",
    }
  }>
  <div
    className="content">
    <PageHeading
      value="About page">
      <h1>
        About page
      </h1>
    </PageHeading>
  </div>
</About>
`;
